File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        int counter = 0;
        vector<int> ans(nums.size()*2);

        for (int i = 0;i < nums.size();i++, counter++)
        {
            ans[i] = nums[i];
        }

		for (int i = 0;i < nums.size();i++, counter++)
		{
			ans[counter] = nums[i];
		}
       

        return ans;
    }
};

File: Data Structures & Algorithms/search-insert-position/submission-0.py
class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:
        mid=len(nums)//2
        if not nums:
            return 0

        if nums[mid]==target:
            return nums.index(target)
        if nums[mid]<target:
            return mid + 1 + self.searchInsert(nums[mid+1:],target)
        else:
            return self.searchInsert(nums[:mid],target)
